### PR TITLE
Used for successful PNO-CCSD calculations

### PR DESCRIPTION
**Pull request by bimalgaudel**

### CODE REVIEW

File: external/versions.cmake [hunk 1]
```diff
@@ -6,8 +6,8 @@ set(SEQUANT_TRACKED_VGCMAKEKIT_TAG 76773f375c7c429ce57d034512bc8e8ffc4c66c8)
 set(SEQUANT_TRACKED_RANGEV3_TAG 0.12.0)
 set(SEQUANT_TRACKED_RANGEV3_PREVIOUS_TAG d800a032132512a54c291ce55a2a43e0460591c7)
 
-set(SEQUANT_TRACKED_TILEDARRAY_TAG 6a134f31f38a1e643583d5c8f19ed85fb61b04ae)
-set(SEQUANT_TRACKED_TILEDARRAY_PREVIOUS_TAG 924c15fc57058b8d9a4dd4a41b5e1e8736c2e339)
+set(SEQUANT_TRACKED_TILEDARRAY_TAG 5cafb3f18d413f3c1313ccd33d5f1570bd6f7553)
```
Comment (evaleev) [inline external/versions.cmake:9]: make sure this is most up to date

**Review by evaleev — CHANGES_REQUESTED** [review]

File: SeQuant/domain/eval/eval.hpp [hunk 2]
```diff
@@ -344,6 +344,7 @@ auto evaluate(NodesT const& nodes, Le const& le, Args&&... args) {
 /// \see EvalResult to know more about the return type.
 ///
 template <typename NodeT, typename Annot, typename Le, typename... Args,
+          std::enable_if_t<IsEvaluable<NodeT>, bool> = true,
           std::enable_if_t<IsLeafEvaluator<NodeT, Le>, bool> = true>
```
Comment (evaleev) [inline SeQuant/domain/eval/eval.hpp:348]: `std::enable_if_t<CondA>, std::enable_if_t<CondB>` is more compactly expressed as `std::enable_if_t<CondA && CondB>`

File: external/versions.cmake (hunk 1 above)
Comment (evaleev) [reply external/versions.cmake:9]: the most recent TA tag is `c10f8d5cade2e6040d28958f04189d6ae578c00a`

File: SeQuant/domain/eval/eval.hpp (hunk 2 above)
Comment (bimalgaudel) [reply SeQuant/domain/eval/eval.hpp:348]: Separating these checks allows identifying the offending check by looking at the compiler error messages when no candidate function is found. That is, in this case, whether `IsEvaluable<NodeT>` failed or `IsLeafEvaluator<NodeT, Le>` failed. Combined, the error message does not tell which check failed. (PS. Error messages articulation might be a compiler specific behavior.)

**Review by evaleev — APPROVED** [review]

**Pull request merged**